### ISSUE TITLE
even when no regions are selected, `atlas.selected_region` is defined

### ISSUE THREAD

**Issue by xgui3783** [FZJ-INM1-BDA/siibra-python#38]
hash/version: e155a16beebdef9b5ca0219168ba0a0c7b255a8e
to reproduce:

```python
import siibra as sb
atlas=sb.atlases.MULTILEVEL_HUMAN_ATLAS
assert atlas.selected_region is None 
```

expected result: assertion passes

actual result: assertion fails. `selected_region` seems to default to selected parcellation

**Comment by dickscheid** [comment]
This is by design. If none is specifically selected, the root node of the parcellation is selected, so selected_region refers to the whole brain (which of course is also a region). I think this is logical, no?

**Comment by dickscheid** [comment]
If we would cahnge this bevahiour, we would need a lot of if/else constructs in the code only to distinguish this case which is in my opinion artificial. Any node of a region tree is a selectable brain region, and so is the whole brain.

**Comment by xgui3783** [comment]
Agreed. I will self assign this issue, and when all above are achieved, I will close this issue.

**Comment by xgui3783** [comment]
new issue raised, redundant code removed (in PR), closing this issue.
